harvester/DBInterface.py

```diff
@@ -580,7 +580,7 @@ def write_record(self, record, repo):
                 elif existing_record["local_identifier"] != record["identifier"]:
                     modified_upstream = True
                 try:
-                    if existing_record["files_size"] != record["files_size"]:
+                    if existing_record["files_size"] != record.get("files_size"):
```

Comment:
Should this default to 0 (e.g. record.get("files_size", 0))?

Reply:
Yes, it probably should have a default value of 0, just in case.